vello/src/wgpu_engine.rs

```diff
@@ -38,7 +38,7 @@ pub(crate) struct WgpuEngine {
     #[cfg(not(target_arch = "wasm32"))]
     shaders_to_initialise: Option<Vec<UninitialisedShader>>,
     pub(crate) use_cpu: bool,
-    /// Overrides from a specific `Image`'s [`id`](peniko::Image::id) to a wgpu `Texture`.
+    /// Overrides from a specific `Image`'s [`id`](peniko::Blob::id) to a wgpu `Texture`.
```

Comment:
Feel like this might be better phrased to indicate that the `id` comes from the image's blob ...